.circleci/config.yml

```diff
@@ -619,6 +619,56 @@ jobs:
       - <<: *STEP_COMPOSER_CACHE_SAVE
       - <<: *STEP_PERSIST_TO_WORKSPACE
 
+  internal_integrations:
+    parameters:
+      docker_image:
+        type: string
+      valgrind_config:
+        type: string
+        default: musl
```

Comment:
This param is not set in the workflow and defaults to musl, while the image is a buster image. Is it a copy paste left over?

Reply:
Nice catch. This is copy pasta.